mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -218,27 +227,71 @@ def create(self, vals):
 
     @api.multi
     def message_auto_subscribe(self, updated_fields, values=None):
-        """Add the responsible, manager and OpenChatter follow list."""
+        """
+        Add the responsible, manager and author to the OpenChatter follow list
+        """
         self.ensure_one()
         user_ids = [
             self.responsible_user_id.id,
             self.manager_user_id.id,
             self.author_user_id.id,
         ]
         self.message_subscribe_users(user_ids=user_ids, subtype_ids=None)
```

Comment:
You're passing as `user_ids` a list of users instead of a recordset. Chances are that this won't work.
I sugges to just make three calls to `message_subscribe_users()`